var/spack/repos/builtin/packages/acts-core/package.py

```diff
@@ -79,4 +77,9 @@ def cmake_variant(cmake_label, spack_variant):
             cmake_variant("MATERIAL_PLUGIN", "material"),
             cmake_variant("TGEO_PLUGIN", "tgeo")
         ]
+
+        if spec.satisfies('+tgeo'):
```

Comment:
So, this is not _quite_ what I want.  This criterion only considers a way through which ACTS might directly link against ROOT, but the ACTS package may also have indirect ROOT dependencies, e.g. via its optional DD4hep support.

Is there a way to interrogate the spec object in order to tell if the active package has a dependency on another package?

Reply:
@HadrienG2 if you want to know if `X` depends on `Y` (in the current DAG, e.g. if Y is a conditional dependency) you can check `if Y in X`. Does that answer your question?

Reply:
It may well answer it, depending on what can be used as `X` and `Y`. Can I do something like `if 'root' in spec`?

Reply:
Yes

Reply:
Let's try this!